src/sig/sig.h

```diff
@@ -242,7 +242,7 @@ typedef struct OQS_SIG {
 	 * compile-time macros `OQS_SIG_*_length_*`.
 	 *
 	 * @param[out] signature The signature on the message represented as a byte string.
-	 * @param[out] signature_len The length of the signature.
+	 * @param[out] signature_len The actual length of the signature. May be smaller than `length_signature` for some algorithms.
```

Comment:
```suggestion
	 * @param[out] signature_len The actual length of the signature. May be smaller than `length_signature` for some algorithms since some algorithms have variable length signatures.
```